backend/typescript/seeders/20211114043430-seed-donations.ts

```diff
@@ -90,12 +86,8 @@ export const up: Seeder = async ({ context: sequelize }) => {
 };
 export const down: Seeder = async ({ context: sequelize }) => {
   await sequelize.getQueryInterface().bulkDelete("scheduling", {
-    id: seedSchedules.map((schedule) => schedule.id),
+    id: [1, 2, 3],
   });
-  await sequelize
-    .getQueryInterface()
-    .bulkDelete("donors", { id: seedDonors.map((donor) => donor.id) });
-  await sequelize
-    .getQueryInterface()
-    .bulkDelete("users", { id: seedUsers.map((user) => user.id) });
+  await sequelize.getQueryInterface().bulkDelete("donors", { id: [1] });
+  await sequelize.getQueryInterface().bulkDelete("users", { id: [1] });
```

Comment:
This is not the most robust seeding logic since it will give errors if we seed after adding some other data. It will only work properly if its seeded right after creating the tables with no data. We can look at ways in making this better later on.